internal/storage/db/internal/db.go

```diff
@@ -349,25 +382,29 @@ func (s *dbStorage) Delete(ctx context.Context, ids ...namer.ModuleID) error {
 	return nil
 }
 
-func (s *dbStorage) GetPolicies(ctx context.Context) ([]*policy.Wrapper, error) {
-	res, err := s.db.From(PolicyTbl).Executor().ScannerContext(ctx)
+func (s *dbStorage) ListPolicyIDs(ctx context.Context) ([]string, error) {
+	res, err := s.db.Select(goqu.C(PolicyTblKindCol), goqu.C(PolicyTblNameCol), goqu.C(PolicyTblVerCol)).
+		From(PolicyTbl).Executor().ScannerContext(ctx)
 	if err != nil {
-		return nil, fmt.Errorf("could not execute %q query: %w", "GetPolicies", err)
+		return nil, fmt.Errorf("could not execute %q query: %w", "ListPolicyIDs", err)
 	}
 	defer res.Close()
 
-	var policies []*policy.Wrapper
+	var policyKeys []string
 	for res.Next() {
 		var rec Policy
 		if err := res.ScanStruct(&rec); err != nil {
 			return nil, fmt.Errorf("could not scan row: %w", err)
 		}
 
-		p := policy.Wrap(rec.Definition.Policy)
-		policies = append(policies, &p)
+		policyKey := fmt.Sprintf("%s.%s", strings.ToLower(rec.Kind), rec.Name)
+		if rec.Version != "" {
+			policyKey += fmt.Sprintf(".%s", rec.Version)
+		}
+		policyKeys = append(policyKeys, policyKey)
```

Comment:
I think we should be able to do the concatenation in SQL itself and remove the extra work done here to scan the records into a policy struct and then concatenate the fields. It'd be much more efficient to scan the result set into a string array directly and it won't generate as much garbage for the GC to deal with either.